src/tests/ftest/pool/destroy_tests.py

```diff
@@ -322,7 +322,6 @@ def test_destroy_wrong_group(self):
                 self.pool.uuid, group_names[0]),
             False)
 
-    @skipForTicket("DAOS-2741")
     def test_destroy_connected(self):
         """Destroy pool with connected client.
```

Comment:
I updated to this PR and ran the test and verified that the pool does not get destroyed when connected.  When trying to destroy the connected pool DAOS reported the expected error as DER_BUSY.

Attempting to destroy a connected pool
Expected exception - destroying connected pool: Pool destroy returned non-zero. RC: -1012

Reply:
oh, In PR same  test is passing [https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-1670/3/artifact/Functional/job-2020-01-22T17.20-55d3c15-pool-destroy_tests/job.log](https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-1670/3/artifact/Functional/job-2020-01-22T17.20-55d3c15-pool-destroy_tests/job.log)